.github/workflows/build.yml

```diff
@@ -1138,7 +1048,6 @@ jobs:
     name: Create Github release
     runs-on: ubuntu-latest
     needs:
-      - archlinux
```

Comment:
I just realized. With that change, we will NOT build archlinux binaries for ArchLinux upon release!


Maybe we can keep it all in (not creating a secondary CI pipeline) and make some individual jobs conditional. What do you think?
I think it should go something like this:

```yaml
jobs:
    archlinux:
        if: github.ref == 'refs/heads/master'
        runs-on: ubuntu-latest
```

that should do the trick. this way we can still keep it all in one build CI pipeline without potentially complicating release pipeline again :)

Reply:
Oh, i didn't know that this is possible :( 
will update PR now